lib/lotus/repository.rb

```diff
@@ -661,6 +666,16 @@ def exclude(query)
         query.negate!
         query
       end
+
+      # This is a method to check entity persited or not
+      # 
+      # @param entity
+      #
+      # @return true
+
+      def persisted?(entity)
+        return true if entity.id
```

Comment:
To ensure the returned value a consistent boolean, you can do this simple trick `!!entity.id`